src/PCIMapper.cpp

```diff
@@ -35,16 +36,13 @@ const PCIDevice& PCIVendor::operator[](const std::string& device_id) const {
 }
 
 // _____________________________________________________________________________________________________________________
-PCIMapper::PCIMapper(const std::string& pci_ids_file) {
-  std::ifstream f_pciid(pci_ids_file);
-  if (!f_pciid) {
-    throw std::runtime_error("ERROR: Could not open file '" + pci_ids_file + "'.\n");
-  }
+PCIMapper::PCIMapper() {
+  std::stringstream s_pciid(get_pcs_ids());
```

Comment:
this should be get_pci_ids(), shouldn't it?

Reply:
Yeah, my bad, sorry. It's fixed now :)